python/cuml/dask/linear_model/logistic_regression.py

```diff
@@ -31,7 +32,8 @@
 np = cpu_only_import("numpy")
 
 
-class LogisticRegression(BaseEstimator, SyncFitMixinLinearModel):
+# class LogisticRegression(BaseEstimator, SyncFitMixinLinearModel):
```

Comment:
Why remove these two mixins?

Reply:
The mixins have been included in the cuml.dask.linear_model.LinearRegression. 

The next line LogisticRegression(LinearRegression) automatically inherits the two mixins, and reuses the code/functions of the LinearRegression class.